src/js/components/mon-compte/export-credentials-component.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { InputLabel, Select, MenuItem } from '@material-ui/core';
+
+class ExportCredentialsField extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = { value: 'python' };
+		this.credentials = props.credentials;
+	}
+
+	handleChange = (e) => this.setState({ value: e.target.value });
+
+	render() {
+		const pythonText = `ceci est l'export en Python : ${this.credentials.AWS_ACCESS_KEY_ID}`;
+		const RText = `ceci est l'export en R : ${this.credentials.AWS_S3_ENDPOINT}`;
+
+		const exportText = this.state.value === 'python' ? pythonText : RText;
+
+		return (
+			<>
+				<InputLabel>Copier au format :</InputLabel>
```

Comment:
If you have time, please consider using I18N (see #21) to translate in both french and english.